packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -859,6 +859,7 @@ class _PopupMenuRoute<T> extends PopupRoute<T> {
     required this.clipBehavior,
     super.settings,
     this.popUpAnimationStyle,
+    this.requestFocus = false,
```

Comment:
Are we sure that this is the behavior that most people want by default?

Reply:
Also, am I right that this parameter will never be passed in since it's private, and it will always be false?

Reply:
I've made changes here, now it defaults to the same behavior as before.